.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "lightgbm-sys/lightgbm"]
 	path = lightgbm-sys/lightgbm
-	url = https://github.com/vaaaaanquish/LightGBM.git
+	url = https://github.com/microsoft/LightGBM.git
```

Comment:
@necnec Thx.

I don't think it's a good practice to use `microsoft/LightGBM`, because many useless files are included in target of build. I removed them in `vaaaaanquish/LightGBM`.
(commit log: https://github.com/vaaaaanquish/LightGBM/commits/master)

We have 2 way.
1. Rebase `microsoft/LightGBM` to `vaaaaanquish/LightGBM` as `vendering`.
    - current way
2. Create a way to exclude useless files from the latest build of `microsoft/LightGBM`.
    - use `exclude/include` in cargo.toml ([doc](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields))
    - improve build.rs